compiler/src/test/java/dagger/internal/codegen/GraphValidationTest.java

```diff
@@ -2131,6 +2268,285 @@ public void subcomponentBindingConflictsWithParentWithNullableViolationAsWarning
+            "import javax.annotation.Generated;",
+            "import javax.inject.Provider",
+            "",
+            GENERATED_ANNOTATION,
+            "public final class DaggerParent implements Parent {",
+            "  private Provider<Set<Object>> setOfObjectProvider;",
+            "  private MembersInjector<Injected> injectedMembersInjector;",
+            "",
+            "  private DaggerParent(Builder builder) {",
+            "    assert builder != null;",
+            "    initialize(builder);",
+            "  }",
+            "",
+            "  public static Builder builder() {",
+            "    return new Builder();",
+            "  }",
+            "",
+            "  public static Parent create() {",
+            "    return new Builder().build();",
+            "  }",
+            "",
+            "  @SuppressWarnings(\"unchecked\")",
+            "  private void initialize(final Builder builder) {",
+            "    this.setOfObjectProvider = SetFactory.<Object>builder(1, 0)",
+            "        .addProvider(Parent_ParentModule_ObjectFactory.create())",
+            "        .build();",
+            "    this.injectedMembersInjector = Injected_MembersInjector",
+            "        .create(setOfObjectProvider);",
+            "  }",
+            "",
+            "  @Override",
+            "  public void inject(Injected injected) {",
+            "    injectedMembersInjector.injectMembers(injected);",
+            "  }",
+            "",
+            "  @Override",
+            "  public Child child() {",
+            "    return new ChildImpl();",
+            "  }",
+            "",
+            "  public static final class Builder {",
+            "    private Builder() {}",
+            "",
+            "    public Parent build() {",
+            "      return new DaggerParent(this);",
+            "    }",
+            "",
+            "    @Deprecated",
+            "    public Builder parentModule(Parent.ParentModule parentModule) {",
+            "      Preconditions.checkNotNull(parentModule);",
+            "      return this;",
+            "    }",
+            "  }",
+            "",
+            "  private final class ChildImpl implements Child {",
+            "    private Provider<Set<Object>> setOfObjectProvider;",
+            "    private MembersInjector<Injected> injectedMembersInjector;",
+            "",
+            "    private ChildImpl() {",
+            "      initialize();",
+            "    }",
+            "",
+            "    @SuppressWarnings(\"unchecked\")",
+            "    private void initialize() {",
+            "      this.setOfObjectProvider = SetFactory.<Object>builder(2, 0)",
+            "          .addProvider(Parent_ParentModule_ObjectFactory.create())",
+            "          .addProvider(Child_ChildModule_ObjectFactory.create())",
+            "          .build();",
+            "",
+            "      this.injectedMembersInjector = Injected_MembersInjector",
```

Comment:
Ok this is good. As expected, a new members injector is created here because there are updates to the multibindings